app/move/index.js

```diff
@@ -1,10 +1,14 @@
 // NPM dependencies
-const router = require('express').Router()
+const express = require('express')
+const router = express.Router()
+const moveRouter = express.Router()
```

Comment:
Is it an idea to, er, move the moveRouter in to a separate module?

Reply:
Yes...I almost went down that rabbit hole but tried to stop myself making this PR about too many different things.

I think it might be worth looking at the approach I've taken with nested apps within the `app/person-escort-record/app/` folder to see how we could make this area a bit better to maintain in the long run.